metricbeat/docs/running-on-kubernetes.asciidoc

```diff
@@ -62,20 +64,62 @@ may want to change that behavior, so just edit the YAML file and modify them:
   value: changeme
 ------------------------------------------------
 
+[float]
+===== Red Hat OpenShift configuration
+
+If you are using Red Hat OpenShift, you need to specify additional settings in
+the manifest file and enable the container to run as privileged.
+
+// REVIEWERS: I wanted to make it clearer as to where this info needs to go. Is
+// this correct? 
```

Comment:
Looks correct to me, maybe we could put here the whole `kubernetes.yml`, so it is more clear how it is expected to be at the end.
Also I think it'd be useful to add it too, commented out, to the deployment files.